docs/topics/proc_add-debug-logging-to-app-vertx.adoc

```diff
@@ -0,0 +1,21 @@
+
+[#_add_debug_logging_to_application_vertx]
+= Adding log output to your {VertX} application.
+
+. To add logging for your application, define a `io.vertx.core.logging.Logger` class:
```

Comment:
You're not defining a class. You're instantiating it. I'd just remove the word "class" from the end of the sentence. Perhaps replace "define" with "create":

>  +. To add logging for your application, create a `io.vertx.core.logging.Logger`:

Reply:
Thank you, re-worded according to your suggestions.